.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+set -eux
+
+TARGETOS=linux
+TARGETARCH=amd64
+
+# Install Kind
+curl -L -o kind "https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-${TARGETARCH}" && chmod +x ./kind
+sudo mv ./kind /usr/local/bin
+
+# Install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl && chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin
+
+# Install helm
+bash -c "curl -s https://get.helm.sh/helm-v3.14.1-linux-${TARGETARCH}.tar.gz > helm3.tar.gz" && tar -zxvf helm3.tar.gz linux-${TARGETARCH}/helm && chmod +x linux-${TARGETARCH}/helm
+sudo mv linux-${TARGETARCH}/helm /usr/local/bin && rm helm3.tar.gz && rm -R linux-${TARGETARCH}
+
+# Install kubebuilder
+curl -L -o kubebuilder "https://go.kubebuilder.io/dl/latest/${TARGETOS}/${TARGETARCH}" && chmod +x kubebuilder
+sudo mv kubebuilder /usr/local/bin/
+
+# Install Github CLI
```

Comment:
Do we need to install github CLI ? Because this is what I see when I ran `gh version` command on forked repo in the codespace.

```
@sunnygoel87 ➜ /workspaces/idpbuilder (main) $ gh version
gh version 2.43.1 (2024-01-31)
https://github.com/cli/cli/releases/tag/v2.43.1
```

Reply:
I'm using devpod with the base image of `"image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",` specify in the devcontainers.json

Reply:
Why I'm including because is useful when testing and reviewing PRs by copy and paste like `gh pr checkout 148`